crates/bevy_render/src/camera/camera.rs

```diff
@@ -391,6 +391,9 @@ pub fn extract_cameras(
             camera.physical_viewport_size(),
             camera.physical_target_size(),
         ) {
+            if target_size.x == 0 && target_size.y == 0 {
```

Comment:
Why is this an AND rather than an OR?

Reply:
Presumably we don't care about rendering anything to a 0 width window regardless of its height.

Reply:
Good catch. This should be or